lib/kitchen/driver/azurerm.rb

```diff
@@ -746,6 +768,26 @@ def get_resource_group
         resource_group
       end
 
+      # Checks whether a resource group exists.
+      #
+      # @param resource_group_name [String] The name of the resource group to check.
+      # The name is case insensitive.
+      #
+      # @return [Boolean] operation results.
+      #
+      def resource_group_exists(resource_group_name)
```

Comment:
If this returns a bool it should end with ?

Reply:
Not sure I understand your suggestion. The function returns the last value and that type is a Boolean type.

Reply:
Yeah it's just a ruby convention to have true/false methods named with ? so it's obvious when reading the code what would be returned by the method

Reply:
Changed.